twindle-cli/twitter/output/twitter-api-response.json

```diff
@@ -1 +1,56 @@
\ No newline at end of file
+{
+  "data": [
```

Comment:
This file got changed when I ran the code. Not sure if this was intended or not.

Reply:
@tolgaerdonmez had removed calling the write to this json file but @PuruVJ added that code back - not sure if this is needed - we will see down the road